corehq/apps/export/templates/export/partial/new_customize_export_templates.html

```diff
@@ -180,6 +195,18 @@ <h4>{% trans "Choose the fields you want to export." %}</h4>
                             </div>
                             <!-- /ko -->
                         </td>
+                        {% if request|toggle_enabled:"EXPORT_MULTISORT" %}
+                        <td class="text-center">
+                            <span class="send-to-bottom">
+                                <i class="fa fa-arrow-down"></i>
```

Comment:
Could I advocate for using "[fa-angle-double-down](https://fontawesome.com/icons/angle-double-down?style=solid)" (and "[fa-angle-double-up](https://fontawesome.com/icons/angle-double-up?style=solid)") to suggest that this moves rows to the top and the bottom instead of just one row up and down? I know the column header makes this clear. But who reads words? :upside_down_face:

Reply:
seconded

Reply:
updated